backend-models/service-tests/accessedservice-tests.txt

```diff
@@ -0,0 +1,129 @@
+
+accessedservice.php
+
+-------------------
+
+********
+* pre-req
+********
+
+(checklogin() is TRUE && (hasAccess($userid, $cid, 'w') || isSuperUser($userid,) is TRUE))
+
+--------------------
+
+Test 1: try to update a created user by changing the username
+Row: 62 - 80 and 220 - 238
```

Comment:
only one service at a time. if the same service is used at another point in the file, then write the related test information there again.